bokeh/models/css.py

```diff
@@ -0,0 +1,375 @@
+    font_variant_east_asian = Nullable(String)
+    font_variant_ligatures = Nullable(String)
+    font_variant_numeric = Nullable(String)
+    font_variant_position = Nullable(String)
+    font_weight = Nullable(String)
+    gap = Nullable(String)
+    glyph_orientation_vertical = Nullable(String)
+    grid = Nullable(String)
+    grid_area = Nullable(String)
+    grid_auto_columns = Nullable(String)
+    grid_auto_flow = Nullable(String)
+    grid_auto_rows = Nullable(String)
+    grid_column = Nullable(String)
+    grid_column_end = Nullable(String)
+    grid_column_gap = Nullable(String)
+    grid_column_start = Nullable(String)
+    grid_gap = Nullable(String)
+    grid_row = Nullable(String)
+    grid_row_end = Nullable(String)
+    grid_row_gap = Nullable(String)
+    grid_row_start = Nullable(String)
+    grid_template = Nullable(String)
+    grid_template_areas = Nullable(String)
+    grid_template_columns = Nullable(String)
+    grid_template_rows = Nullable(String)
+    height = Nullable(String)
+    hyphens = Nullable(String)
+    image_orientation = Nullable(String)
+    image_rendering = Nullable(String)
+    inline_size = Nullable(String)
+    justify_content = Nullable(String)
+    justify_items = Nullable(String)
+    justify_self = Nullable(String)
+    left = Nullable(String)
+    letter_spacing = Nullable(String)
+    lighting_color = Nullable(String)
+    line_break = Nullable(String)
+    line_height = Nullable(String)
+    list_style = Nullable(String)
+    list_style_image = Nullable(String)
+    list_style_position = Nullable(String)
+    list_style_type = Nullable(String)
+    margin = Nullable(String)
+    margin_block_end = Nullable(String)
+    margin_block_start = Nullable(String)
+    margin_bottom = Nullable(String)
+    margin_inline_end = Nullable(String)
+    margin_inline_start = Nullable(String)
+    margin_left = Nullable(String)
+    margin_right = Nullable(String)
+    margin_top = Nullable(String)
+    marker = Nullable(String)
+    marker_end = Nullable(String)
+    marker_mid = Nullable(String)
+    marker_start = Nullable(String)
+    mask = Nullable(String)
+    mask_composite = Nullable(String)
+    mask_image = Nullable(String)
+    mask_position = Nullable(String)
+    mask_repeat = Nullable(String)
+    mask_size = Nullable(String)
+    mask_type = Nullable(String)
+    max_block_size = Nullable(String)
+    max_height = Nullable(String)
+    max_inline_size = Nullable(String)
+    max_width = Nullable(String)
+    min_block_size = Nullable(String)
+    min_height = Nullable(String)
+    min_inline_size = Nullable(String, help="""
+    The `min-inline-size`_ CSS property defines the horizontal or vertical
+    minimal size of an element's block, depending on its writing mode. It
+    corresponds to either the ``min-width`` or the ``min-height`` property,
+    depending on the value of ``writing-mode``.
+
+    .. _min-inline-size: https://developer.mozilla.org/en-US/docs/Web/CSS/min-inline-size
```

Comment:
Why a help string for just this one single property? Maybe better to just leave this off and have a single link at the top level to a CSS reference?

Reply:
As indicated in the commit message, this is an experiment. I added one to show how it would look like, and to test whether sphinx will accept this (I never know what to expect for rst/sphinx, as it seems to fail often for no good reason). Besides this, adding all this manually is out of question.